cfme/fixtures/provider.py

```diff
@@ -216,7 +216,7 @@ def setup_one_or_skip(request, filters=None, use_global_filters=True):
         if _setup_provider_verbose(request, provider):
             return provider
 
-    skip_msg = "Failed to set up any matching providers: {}", [p.key for p in providers]
+    skip_msg = "Failed to set up any matching providers: {}".format([p.key for p in providers])
```

Comment:
I'd vote for an f-string here! I think @mshriver went and changed all the `format`'s to f-string's not too long ago. 

```suggestion
    skip_msg = f"Failed to set up any matching providers: {[p.key for p in providers]}."
```

Reply:
Updated all uses of format() to be f-strings instead.